app/views/authors/show.html.haml

```diff
@@ -0,0 +1,11 @@
+.author
+  .name
+    = @author.full_name
+  .biography= @author.biography
+  %ul.author-books
```

Comment:
@Fatpaher I think it will be better to use class `books` instead of `author`, because it is already nested in `author` div.